mne/forward/forward.py

```diff
@@ -1138,21 +1140,23 @@ def apply_forward(fwd, stc, evoked_template, start=None, stop=None,
     --------
     apply_forward_raw: Compute sensor space data and return a Raw object.
     """
+    if evoked_template is not None or not isinstance(info, Info):
+        warnings.warn('"evoked" is deprecated and will be removed in '
+                      'MNE-0.11. Please give evoked.info instead',
+                      DeprecationWarning)
+        info = info.info
```

Comment:
this is incorrect if `evoked_template is not None`, so what you use for `info` needs to be triaged here